cmd/azure.go

```diff
@@ -38,9 +39,35 @@ import (
 
 const globalAzureAPIVersion = "2016-05-31"
 
+// To store metadata during NewMultipartUpload which will be used after
+// CompleteMultipartUpload to call SetBlobMetadata.
+type azureMultipart struct {
```

Comment:
will call it `azureMultipartMetaInfo` but it looks long for a type, calling it `azureMultipartInfo` will be incorrect as it does not hold info related to multipart but rather meta of the multipart.